etc/kayobe/inventory/group_vars/all/dev-tools

```diff
@@ -0,0 +1,8 @@
+---
+########################################################################
+# List of development tool packages to install during host configuration
+dev_tools_packages_extra:
+  - tcpdump
+  - jq
+  - wget
+  - htop
```

Comment:
`htop` is only in EPEL for Rocky Linux 9. You could make it depend on distribution is Ubuntu or EPEL is enabled.

Reply:
Done